src/transformers/modeling_utils.py

```diff
@@ -2006,7 +2006,16 @@ def save_pretrained(
                         peft_state_dict[f"base_model.model.{key}"] = value
                     state_dict = peft_state_dict
 
-                current_peft_config = self.peft_config[self.active_adapter()]
+                active_adapter = self.active_adapters()
+
+                if len(active_adapter) > 1:
+                    logger.warning(
```

Comment:
Instead of warning, I would raise here. It's important for the user to understand the difference between multiple active adapters and single. I would limit the functionality for multi active adapters heavily for now